velox/functions/sparksql/Hash.cpp

```diff
@@ -26,21 +26,273 @@ namespace {
 
 const int32_t kDefaultSeed = 42;
 
+struct Murmur3Hash;
+struct XxHash64;
+
+/// A template struct that contains the seed and return type of the hash
+/// function.
```

Comment:
nit: /// is used for multi-line comments which are to be exposed. In a anonymous namespace, // is needed.